numpy/core/_internal.py

```diff
@@ -631,4 +631,17 @@ class TooHardError(RuntimeError):
     pass
 
 class AxisError(ValueError, IndexError):
-    pass
+    """ Axis supplied was invalid. """
```

Comment:
You can import from practically anything. We also import from multiarray itself into C code. I'd import this from '"numpy"', The `_globals` is for singletons that may also be used by python code.